app/components/concept-page/question-card-option.hbs

```diff
@@ -23,7 +23,7 @@
       <div class="rounded-full w-3 h-3 bg-gray-500"></div>
     {{/if}}
   </div>
-  <div class="prose {{if (and this.isSubmitted this.isCorrect) 'text-gray-800 font-semibold' 'text-gray-800'}}">
+  <div class="prose text-left {{if (and this.isSubmitted this.isCorrect) 'text-gray-800 font-semibold' 'text-gray-800'}}">
```

Comment:
@libmartinito can we add this to `button` instead? I think that's what causes the override in the first place (a `prose` _should_ be text-left by default?)